query-service-impl/src/test/java/org/hypertrace/core/query/service/pinot/PinotBasedRequestHandlerTest.java

```diff
@@ -141,6 +143,119 @@ public void testCanHandle() {
     }
   }
 
+  @Test
+  public void testApplyAdditionalFilters() {
+    for (Config config : serviceConfig.getConfigList("queryRequestHandlersConfig")) {
+      if (!isPinotConfig(config)) {
+        continue;
+      }
+
+      PinotBasedRequestHandler handler =
+          new PinotBasedRequestHandler(
+              config.getString("name"), config.getConfig("requestHandlerInfo"));
+
+      // domain-service-handler only has filter and no time range
+      if (config.getString("name").equals("domain-service-handler")) {
```

Comment:
Is this test using the correct handler - `domain-service-handler`? For this, the original request should be the same after applying filters, right?

Reply:
I think, it should have been - `trace-view-handler`, right?

Reply:
domain-event-view only has filter.
span-event-view only has time range 
trace-view has both time range and filter.